dask/base.py

```diff
@@ -1250,7 +1251,13 @@ def get_scheduler(get=None, scheduler=None, collections=None, cls=None):
             return scheduler.get
         elif isinstance(scheduler, str):
             scheduler = scheduler.lower()
+
             if scheduler in named_schedulers:
+                if config.get("scheduler", None) in ("dask.distributed", "distributed"):
+                    warnings.warn(
+                        "Passing a local execution scheduler in Dask.distributed might "
+                        "lead to unexpected results."
```

Comment:
I think this wording is potentially confusing. How about:

```suggestion
                        "Running on a single-machine scheduler when a distributed client "
                        "is active might lead to unexpected results."
```

Reply:
Hi @jsignell , I think your wording is better and I will updated it in my next PR.